documentation/asciidoc/topics/proc_disabling_encryption.adoc

```diff
@@ -0,0 +1,22 @@
+[id='disabling_encryption-{context}']
+= Disabling Encryption
+You can disable encryption so clients do not need TLS certificates to establish connections with {brandname}.
```

Comment:
maybe it worth to move this section near the `Openshift Service Certificates`? I mean: on plain k8s, connection is unencrypted by default unless certs are explicitly provided

Reply:
sounds good @rigazilla I'll update the docs now